packages/core/src/components/ScreenContainer.tsx

```diff
@@ -14,18 +14,18 @@ import type { Theme } from "../styles/DefaultTheme";
 type Props = {
   hasSafeArea: boolean;
   scrollable: boolean;
-  hasBottomSafeArea?: boolean;
-  hasTopSafeArea?: boolean;
+  hasBottomSafeArea: boolean;
+  hasTopSafeArea: boolean;
   theme: Theme;
   style?: StyleProp<ViewStyle>;
   children?: React.ReactNode;
 };
 
 function ScreenContainer({
-  hasSafeArea = false,
+  hasSafeArea,
```

Comment:
If you're removing the default for `hasSafeArea`, is the type `boolean` correct? It won't always be supplied by the builder, right? I think perhaps it should be `hasSafeArea?: boolean;`

Reply:
@dhruska I made it required with a default prop.  My feeling is if you use a default prop then you should not make it optimal.